Automattic/jetpack
# General: Update module headings

Updates the Lazy Images module description. Follow-up to #9840 
#### Changes proposed in this Pull Request:

* Updates `modules/module-headings` to include the changes from #9840 

#### Testing instructions:

Confirm that the change matches the lazy images description in `modules/lazy-images.php`
